uhabits-android/src/androidTest/java/org/isoron/uhabits/activities/habits/list/views/NumberButtonViewTest.kt

```diff
@@ -96,7 +96,7 @@ class NumberButtonViewTest : BaseViewTest() {
     fun testClick_shortToggleDisabled() {
         prefs.isShortToggleEnabled = false
         view.performClick()
-        assertFalse(edited)
+        assertTrue(edited)
     }
```

Comment:
Since any type of click now opens a dialog, it's no longer necessary to have `testClick_shortToggleDisabled` and `testClick_shortToggleEnabled`. We can just have `testClick`.